subprojects/core-api/src/main/java/org/gradle/caching/local/DirectoryBuildCache.java

```diff
@@ -28,7 +28,7 @@
  */
 public abstract class DirectoryBuildCache extends AbstractBuildCache {
     private Object directory;
-    private int removeUnusedEntriesAfterDays = 7;
+    private int removeUnusedEntriesAfterDays = -1;
```

Comment:
I suppose this is a breaking change, if anyone is checking the default value programmatically. The effective default value is the same, since this is taken from `CacheConfiguration.createdResources()`, which has a 7 day default.

Reply:
❓ What about if we add some internal property that will return if `removeUnusedEntriesAfterDays` was set or not?

Reply:
This change was reverted: the default value is again `7`.